.github/workflows/delete-packagecloud-packages.yml

```diff
@@ -0,0 +1,38 @@
+name: PackageCloud nightly package delete schedule
+
+env:
+  PACKAGE_CLOUD_API_TOKEN: ${{ secrets.PACKAGE_CLOUD_API_TOKEN }}
+
+on:
+  push:
+    branches:
+      - "**"
```

Comment:
Does it really make sense to run this job on every branch push?

I suggest we remove `on.push` lines, and use only `on.schedule`.

Reply:
Fixed as you suggested